src/Api/Controllers/SendsController.cs

```diff
@@ -64,13 +66,21 @@ public async Task<IActionResult> Access(string id, [FromBody] SendAccessRequestM
         }
 
         [AllowAnonymous]
-        [HttpGet("access/file/{id}")]
-        public async Task<SendFileDownloadDataResponseModel> GetSendFileDownloadData(string id)
+        [HttpGet("{encodedSendId}/access/file/{fileId}")]
```

Comment:
The AccessId used during a Send Access is just a base64Url encoding of the Send's Id. I'm not sure why @kspearrin did this. I think he was worried about Guids containing url-unsafe characters, but I don't think they can. We don't always encode the Id either.

In this case, the Id is encoded because this method is used during Access and that's what we have available to us. It needs to be decoded so we can look up the file's location, since I included the SendId in the blob/file path.

The sendId is included in the blob/file path because we needed a quick way to determine a Send from the blob create event delivered by Azure. The blob path seemed the best way.